docs/my-website/docs/providers/friendli_ai.md

```diff
@@ -0,0 +1,53 @@
+# FriendliAI
+https://friendli.ai/
+
+**We support ALL FriendliAI models, just set `friendli_ai/` as a prefix when sending completion requests**
+
+## API Key
+```python
+# env variable
+os.environ['FRIENDLI_AI_API_KEY']
```

Comment:
If env var name doesn't matter much, I suggest `FRIENDLI_TOKEN` since it's used in [our CLI](https://github.com/friendliai/periflow-client-internal?tab=readme-ov-file#python-sdk-examples)